package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "prebuild": "rimraf dist",
     "build": "nest build",
+    "buildNativeModules": "cargo build --release && ts-node ./native/copyNativeLibs.ts --release ./native/compiled/",
```

Comment:
this compiles everything it finds in our root-most Cargo.toml, then plucks the compiled `.dll/dylib.so`s out of the target folder, and stuffs them all into `/native/compiled`. Would love feedback on folder organization here